velox/exec/tests/SharedArbitratorTest.cpp

```diff
@@ -382,6 +383,173 @@ class SharedArbitrationTest : public exec::test::HiveConnectorTestBase {
     return queryCtx;
   }
 
+  std::shared_ptr<Task> makeHashJoinTask(
```

Comment:
nit: shall we just call runHashJoinTask as it create, execute and verify the task results? Thanks!